src/util/browser.ts

```diff
@@ -504,14 +504,14 @@ export class Browser {
           recorder.swFrameIds.add(frameId);
         }
 
-        if (recorder.swFrameIds && recorder.swFrameIds.has(frameId)) {
+        if (recorder.hasFrame(frameId)) {
           foundRecorder = recorder;
           break;
         }
       }
 
       if (!foundRecorder) {
-        logger.debug(
```

Comment:
not sure if this should be a warning.. means something is getting skipped, but maybe debug is ok for that?